### PR TITLE
css: Move some CSS imports to SCSS

### PR DESCRIPTION
**Pull request by garrett**
This prevents duplication and includes CSS in the correct place.

### CODE REVIEW

**Comment by garrett** [comment]
Rebased.

This needs review.

File: pkg/shell/shell.scss [hunk 1]
```diff
@@ -22,6 +22,7 @@
 @use "page";
 @use "../lib/table.css";
 @use "nav.scss";
+@import "@patternfly/patternfly/components/Select/select.css";
```
Comment (KKoukiou) [inline pkg/shell/shell.scss:25]: Does it make a difference to @use the .scss file instead?

Comment (garrett) [reply pkg/shell/shell.scss:25]: Yes. CSS doesn't support `@use`; that's a SASS thing. (Note the file extension.)

**Review by KKoukiou — COMMENTED** [review]
Please rebase and I am fine with this.

**Review by KKoukiou — DISMISSED** [review]

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**